presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -475,7 +491,19 @@ public CounterStat getHighMemorySplitSource()
 
                 results.add(new HivePartitionMetadata(hivePartition, Optional.of(partition), partitionSchemaDifference.build(), encryptionInformation));
             }
-
+            String prestoWarningMessage = format("Table '%s' has '%s' out of '%s' partitions unreadable: ", tableName, partitionsNotReadable.values().stream().mapToInt(Set::size).sum(), partitionBatch.size());
```

Comment:
- `partitionsNotReadable.values().stream().mapToInt(Set::size).sum()` should be replaced by `unreadablePartitionsSkipped`
- just name `warningMessage`

Reply:
@highker I made a mistake here, should have been unreadablePartitionsSkipped (number of partitions on in partitionsUnreadable) + partitionsNotReadable.values().stream().mapToInt(Set::size).sum()

Does that make sense?